### PR TITLE
Add "warmup"

### PR DESCRIPTION
**Pull request by davidnvq**
Từ warmup này xuất hiện trong phần Learning rate scheduler.
Mình tạm đề xuất là `khởi động`, mượn bên thể thao. Mà vẫn phân vân vì ngày xưa đi trường hè, có thầy Hưng (director VinAI giờ) có trình bày về khởi đầu nguội, rồi có 1 cụm từ `làm nóng` gì đó. Nhớ mang máng vậy.

### CODE REVIEW

**Comment by duythanhvn** [comment]
Cái chỗ ngữ cảnh bổ sung đấy mình đang thấy Bảng thuật ngữ sử dụng cả Anh, Việt hơi lẫn lộn. Anh em muốn giữ như vậy, hay chọn một ngôn ngữ để bổ sung ngữ cảnh thôi; hoặc tiếng Anh, hoặc tiếng Việt?

cc: @minhduc0711 @davidnvq @lkhphuc @rootonchair @cuongvng

**Comment by cuongvng** [comment]
em vote ngữ cảnh tiếng Việt.

**Comment by minhduc0711** [comment]
@duythanhvn Em thì thích tiếng anh hơn, tại đang có sẵn mấy ngữ cảnh như CNN, softmax.

Ngoài ra thì em thấy dấu ngoặc đơn đang hơi bị overload: ngữ cảnh, cách viết tắt, từ optional, loại từ.
Em nghĩ mình nên thống nhất 1 cách format gọn gàng hơn chút (có thể thêm cả cột mới cho context?).

**Comment by davidnvq** [comment]
Thế chắc bổ sung thêm cột Note nữa r điền nhưng thông tin bổ sung đó vào 1 cột phát sinh đó

**Comment by duythanhvn** [comment]
> Thế chắc bổ sung thêm cột Note nữa r điền nhưng thông tin bổ sung đó vào 1 cột phát sinh đó

Ngôn ngữ cho phần ngữ cảnh ông giáo muốn là gì? Ông giáo vote tiếng Anh hay tiếng Việt?

**Comment by lkhphuc** [comment]
> > Thế chắc bổ sung thêm cột Note nữa r điền nhưng thông tin bổ sung đó vào 1 cột phát sinh đó
> 
> Ngôn ngữ cho phần ngữ cảnh ông giáo muốn là gì? Ông giáo vote tiếng Anh hay tiếng Việt?

Mình thấy cột tiếng nào thì chú thích bằng tiếng đấy thôi, như chú thích cho từ `warm-up` thì mình nên thêm bằng tiếng anh, còn chú thích cho cách dịch tiếng việt thì mình mở ngoặc tiếng việt bên phần cách dịch.

**Comment by duythanhvn** [comment]
> Mình thấy cột tiếng nào thì chú thích bằng tiếng đấy thôi, như chú thích cho từ `warm-up` thì mình nên thêm bằng tiếng anh, còn chú thích cho cách dịch tiếng việt thì mình mở ngoặc tiếng việt bên phần cách dịch.

Đề xuất này khá hợp lý. Anh em thấy sao?

File: glossary.md [hunk 1]
```diff
@@ -484,6 +484,7 @@ Nếu bạn cho rằng một từ không nên dịch ra tiếng Việt, bạn c
 ## W
 | English                                   | Tiếng Việt        | Thảo luận tại                                |
 |-------------------------------------------|-------------------|----------------------------------------------|
+| warmup (trong định thời tốc độ học)       | khởi động         | [https://git.io/JJIT5](https://git.io/JJIT5) |
```
Comment (cuongvng) [inline glossary.md:487]: em đồng ý.
```suggestion
| warmup (in learning rate scheduling)       | khởi động (trong định thời tốc độ học)        | [https://git.io/JJIT5](https://git.io/JJIT5) |
```

Comment (lkhphuc) [reply glossary.md:487]: Mình thấy phần `(trong định thời tốc độ học)` chỗ này hơi thừa, không cần thiết lắm. Chú thích bên tiếng việt giống như kiểu `(câu) truy vấn` ấy. Mình mở ngoặc để chú thích linh động nhiều cách dịch, có hay không từ `câu` cũng được.

Comment (davidnvq) [reply glossary.md:487]: em đồng ý.
```suggestion
| warmup (in learning rate scheduling)       | khởi động     | [https://git.io/JJIT5](https://git.io/JJIT5) |
```

**Comment by duythanhvn** [comment]
Hic, ông giáo @davidnvq bình tĩnh nhé, này chưa xong mà.. nhờ em @minhduc0711 kiểm tra lại Bảng thuật ngữ rồi điều chỉnh lại theo thảo luận trong PR này em nha.

File: glossary.md (hunk 1 above)
Comment (cuongvng) [reply glossary.md]: ủa em hiểu là đang thảo luận thêm chú thích ngữ cảnh như nào, chứ không phải chú thích cách dịch.

**Comment by davidnvq** [comment]
À okie, hiểu nhầm là đôi chút :'(

**Comment by minhduc0711** [comment]
@duythanhvn thế cuối cùng là sửa hết về tiếng anh ạ?

**Comment by duythanhvn** [comment]
Ừa, tạm thời thì đưa về tiếng Anh hết hen.
